corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -20,13 +20,10 @@
 {% block js-inline %}{{ block.super }}
     <script type="text/javascript">
         var LOAD_LOCS_URL = '{{ api_root }}';
-
-        // for product and user per location selection
-        $(function () {
-            $("#id_users-selected_ids").width(800).height(400).multiselect();
-            $("#id_products-selected_ids").width(800).height(400).multiselect();
-        });
+            NEW_MOBILE_WORKER_URL = '{% url "new_mobile_worker_modal" domain %}';
+            LOCATION_ID = {{ location.get_id|JSON }};
```

Comment:
Yeah, it's a tricky situation.  In this case there's a 1-1 javascript and html - `location.html` and `location.js`.  The page is primarily synchronous as well, so it's nice to load all the necessary resources at once, rather than making separate ajax requests for initialization data.  We do this pattern from time to time to make dynamic, template-generated data available to javascript, also for strings that need to be translated (you don't want to dynamically generate javascript files, clearly).

Maybe I could rewrite this so the static file defines utils that I bind here in the main template.  What do you think?